logstash-filter-translate.gemspec

```diff
@@ -21,7 +21,15 @@ Gem::Specification.new do |s|
 
   # Gem dependencies
   s.add_runtime_dependency "logstash-core-plugin-api", ">= 1.60", "<= 2.99"
+  s.add_runtime_dependency 'rufus-scheduler'
+
+  if RUBY_VERSION.start_with?("1")
+    s.add_runtime_dependency 'rake', '~> 12.1.0'
+  end
```

Comment:
what is this needed for? I haven't seen that in other plugins?

Reply:
I need to do this when I manually installed the built gem on an existing LS 5.X.X install. 😕 .
Travis does not have this problem as it builds 5.6 from scratch.
The latest file input PR did this too but pinned it at `12.2.0`
I am not sure where the runtime dependency on rake comes from.

Reply:
yeah my problem here is that if we start introducing such litlle differences if every plugin we work on it will become a mess. I really wish we could keep stuff like Rakefile, gemspec, build.gradle, etc as generic as possible. some of these problem are environment dependent and should probably be only fixed locally. when we see such a problem which is not only a local env problem we should create a separate issue which not only solve it for a specific plugin but for all plugins.

Reply:
True.
My concern is that some popular plugins will get installed back in older versions of LS and if we know that it will be a problem and we do nothing it will cause more user pain.

I don't yet know of a smooth answer to this.

Reply:
can we remove that and open an issue to find a proper solution to this problem?

Reply:
Sure.